File: A01Hello/app/src/main/java/com/kbstar/android/MainActivity.java
package com.kbstar.android;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);  // R.layout = res > layout 이라는 뜻!

        /*
            JS
            var btn = document.getElementById('btn');
            let btn = document.querySelector('#btn');

            JQuery
            let btn = $('#btn');
         */

        textView = findViewById(R.id.textView); // 머 찾아올 땐 find!!!
        System.out.println("-------------------" + textView.getText().toString());
    }

}

File: AndroidLab/test1_resource/src/main/java/com/kbstar/test1_resource/MainActivity.kt
package com.kbstar.test1_resource

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)  // 이름이 같으니까 기본/land 알아서 적용됨
    }
}